XSUP-66808 | doc alert aggregation (#43798)
demisto/content: Packs/PrismaCloudCompute/README.md

> If any alerts are missing in Cortex XSIAM, check the status of the integration:

Reviewer: ```suggestion
If any alerts are missing in Cortex XSIAM, do the following.
```

![image](doc_files/prisma_instance.png)

> ### Fewer Alerts in Cortex XSIAM Than in Prisma Cloud Compute

Reviewer: ```suggestion
### Troubleshooting: Mismatch Between Prisma Cloud Alerts and Cortex XSIAM Issues
```

> Prisma Cloud Compute groups alerts that occur close together into a single aggregated alert before sending them to Cortex XSIAM. Each aggregated alert creates one case, even if it contains multiple individual alerts. Your alerts are not lost — they are stored inside each case in the `aggregatedAlerts` field and can be viewed with this XQL query:

Reviewer: Doesn't XSIAM use issues and cases? Also, from the explanation in the integration README, the alerts in Prisma Cloud Compute are aggregated and then become one issue in XSIAM - not case. Please confirm this is the correct explanation. In the meantime, recommend aligning this with the integration README:

```suggestion
Prisma Cloud Compute aggregates alerts occurring within a specific time window into a single alert before sending them to Cortex XSIAM. This may result in fewer issues created compared to the total number of alerts in the Prisma Cloud Compute console. Your alerts are not lost — they are stored inside each issue in the `aggregatedAlerts` field and can be viewed with this XQL query:
```

```xql
dataset = prisma_cloud_compute_raw
| fields aggregatedAlerts
```

> To receive each alert as a separate case, reduce the aggregation window in your Prisma Cloud alert profile:

Reviewer: ```suggestion
To receive each alert as a separate issue, reduce the aggregation window in your Prisma Cloud alert profile:
```

If any alerts are missing in Cortex XSOAR, check the status of the integration:

![image](doc_files/prisma_instance.png)

> ### Fewer Incidents in Cortex XSOAR Than in Prisma Cloud Compute

Reviewer: ```suggestion
### Troubleshooting: Mismatch Between Prisma Cloud Alerts and Cortex XSOAR Incidents
```

> Prisma Cloud Compute groups alerts that occur close together into a single aggregated alert before sending them to Cortex XSOAR. Each aggregated alert creates one incident, even if it contains multiple individual alerts. The individual alerts are preserved in the `aggregatedAlerts` field of each incident.

Reviewer: ```suggestion
Prisma Cloud Compute aggregates alerts occurring within a specific time window into a single alert before sending them to Cortex XSOAR. This may result in fewer incidents created compared to the total number of alerts in the Prisma Cloud Compute console. Your alerts are not lost — they are stored inside each incident in the `aggregatedAlerts` field.
```
